.changeset/famous-hats-shout.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/itwinui-react": minor
```

Comment:
Should be patch?

Reply:
I thought it should be `minor` since when the tooltip is shown has changed. But I can rephrase the changeset message to be like a `fix` if we feel this PR is more of a bug fix than a change in the behavior.

Reply:
Yeah I think this is more of a small bug fix.

`minor` should be used for new features, API changes, and larger refactorings (like the Table scroll container change in #1891).

Reply:
Sounds good, changed to `patch`. Also reworded the message to be a patch changeset. (https://github.com/iTwin/iTwinUI/pull/1944/commits/a820fcf0bd927129a8d627e9901a0780cd6279cf)